src/main/java/org/kainos/ea/exceptions/DatabaseConnectionException.java

```diff
@@ -0,0 +1,9 @@
+package org.kainos.ea.exceptions;
+
+public class DatabaseConnectionException extends Exception {
+  public DatabaseConnectionException(Exception e) {
+    super(e);
+  }
+
+
```

Comment:
Some extra lines

Reply:
I've removed these - not sure why not picked up. Done now.